src/System.Management.Automation/engine/PowerShellStreamType.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace System.Management.Automation
+{
+    /// <summary>
+    /// Enumeration of the possible PowerShell stream types.
+    /// This enumeration is obsolete.
+    /// </summary>
+    /// <remarks>
+    /// This enumeration is a public type formerly used in PowerShell Workflow,
+    /// but kept due to its generic name and public accessibility.
+    /// It is not used by any other PowerShell API, and is now obsolete
+    /// and should not be used if possible.
+    /// </remarks>
+    [Obsolete("Formerly used in PowerShell Workflow. Use an integer or System.Management.Automation.Language.RedirectionStream instead.")]
```

Comment:
> Use an integer or System.Management.Automation.Language.RedirectionStream instead.

@TravisEz13 and I discussed on this message, and we think it's better to not give a workaround here given that those 2 enums have different members and values.

How about change the message to
```
This enum type was used only in PowerShell Workflow and is now obsolete
```
And make sure specify `true` for the `error` parameter.